site/zenodo_rdm/utils.py

```diff
@@ -164,3 +164,54 @@ def swh_link_render(record):
             )
         )
     return ret
+
+
+def blr_link_render(record):
+    """Entry for BLR."""
+    ret = []
+
+    treatmentbank_link = None
+    gbif_link = None
+    sibls_link = None
+
+    for identifier in record["metadata"]["identifiers"]:
+        if "http://publication.plazi.org/id/" in identifier["identifier"] or "http://treatment.plazi.org/id/" in identifier["identifier"]:
```

Comment:
```suggestion
        if re.match("(http|https)://(publication|treatment).plazi.org/id/", identifier["identifier"], re.I):
```

+ ``import re`` in the top